src/components/check-box.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+This is a component for a checkbox with accompanying text.
+Input:
+  label: The text displayed next to the checkbox
+  href: The URL for the hyperlink
+  hrefLabel: The text displayed for the hyperlink
+Output:
+  A checkbox input with associated text that is compatible with react-hook-form
+*/
+
+import {
+  FieldValues,
+  useController,
+  UseControllerProps
+} from "react-hook-form";
+export interface FormProps<T extends FieldValues = FieldValues>
+  extends UseControllerProps<T> {
+  label?: string;
+  href?: string;
+  hrefLabel?: string;
+}
+
+export default function Field_Checkbox<T extends FieldValues = FieldValues>({
+  label,
+  href,
+  hrefLabel,
+  ...props
+}: FormProps<T>) {
+  const { field } = useController({ ...props });
+
+  return (
+    <div className="order-3 flex h-12 w-96 flex-none items-center self-stretch">
```

Comment:
A lot of these classes don't do anything. Be sure to remove what you don't need

Reply:
As a reusable component using fixed width and height will cause problems in the future. If you need some extra spacing, consider `gap`, `padding` and `margins`